src/scss/layout/_footer.scss

```diff
@@ -0,0 +1,116 @@
+.footer{
+    background-color: #577598;
+    display: grid;
+    grid-template-columns: 2fr 1fr 1fr;
+    height: 461px;
+    max-width: 1440px;
```

Comment:
Please define this on a higher level. The max width of the footer is not only on this component, but the whole application